core/src/main/scala/akka/kafka/javadsl/Consumer.scala

```diff
@@ -104,6 +104,15 @@ object Consumer {
       .mapMaterializedValue(new WrappedConsumerControl(_))
       .asJava
 
+  /**
+   * Transactional source to setup a stream for Exactly Only Once (EoS) kafka message semantics.  To enable EoS it's
+   * necessary to use the [[Producer#transactionalSink]] or [[Producer#transactionalFlow]] (for passthrough).
+   */
+  def transactionalSource[K, V](consumerSettings: ConsumerSettings[K, V], subscription: Subscription): Source[TransactionalMessage[K, V], Control] =
+    scaladsl.Consumer.transactionalSource(consumerSettings, subscription)
+      .mapMaterializedValue(new WrappedConsumerControl(_))
+      .asJava
```

Comment:
I expect a type error here as we upgraded to Akka 2.5.12.
Could you rebase, please.